packages/button-react/src/Button.tsx

```diff
@@ -26,54 +26,41 @@ const makeButtonComponent = (buttonType: ValidButtons) => {
                 }
             };
 
-            if (!loader) {
-                return (
-                    <BaseButton
-                        className={classNames(componentClassName, className)}
-                        onClick={onClick}
-                        onTouchStart={handleTouch}
-                        {...rest}
-                        ref={ref}
-                    >
-                        {children}
-                    </BaseButton>
-                );
-            }
-
             return (
-                <div
-                    className={classNames("jkl-button-wrapper", className, {
-                        "jkl-button-wrapper--compact": forceCompact,
-                    })}
+                <BaseButton
+                    className={componentClassName}
+                    onClick={onClick}
+                    onTouchStart={handleTouch}
+                    disabled={loader?.showLoader}
+                    aria-live="polite"
+                    {...rest}
+                    ref={ref}
                 >
                     <div
-                        className={classNames("jkl-button-wrapper__slider", {
-                            "jkl-button-wrapper__slider--show-loader": !!loader.showLoader,
+                        className={classNames("jkl-button-wrapper", {
```

Comment:
Føles litt rart å ha `button-wrapper` inne i `button` 😅 Kanskje heller kalle den `loader-wrapper` eller noe? Er nesten fristende å ta inn `ContentToggle` her, siden den jo gjør akkurat denne jobben. Kommer an på om vi vil ha inn enda en avhengighet på noe så enkelt som en knapp.

Reply:
Den wrapper alt innhold i knappen så `loader-wrapper` blir også litt rart for meg. Enig i at wrappere typisk er på utsiden. `button-content`?

Reply:
Ja stemmer, for nå er alltid wrapperen der og det er bare selve loaderen som fjernes fra DOMet. `button-content` er sikkert fint isf!